lib/Comment.php

```diff
@@ -46,12 +48,17 @@ class Comment extends Core implements CoreInterface {
 	protected $children = array();
 
 	/**
-	 * @param int $cid
+	 * Constructor.
+	 *
+	 * @param int $cid Comment id.
```

Comment:
I’d say when we write about an ID, we always use capital letters: `ID`. This is how WordPress does it in most of the cases.